src/client/src/routes/StyleguideRoute/components/ColorToClipBoard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCopy } from '@fortawesome/free-solid-svg-icons'
+import { styled } from 'rt-theme'
+
+library.add(faCopy)
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+interface ColorToClipBoardProps {
```

Comment:
Naming inconsistency, line 8 the original import kebab-case to PascalCase is correct.  But on line 10 and elsewhere `Clipboard` becomes `ClipBoard` which isn't consistent.  I'd either alter line 8 to match elsewhere be consistent with the original kebab case import.